packages/arb-token-bridge-ui/src/components/common/Notifications.tsx

```diff
@@ -27,8 +27,8 @@ function Notification({
   return (
     <div
       className={twMerge(
-        'mx-2 flex w-auto gap-2 rounded-md p-2 px-4 text-sm',
-        mode === 'light' ? 'bg-cyan text-dark' : 'bg-dark text-cyan'
+        'mx-6 flex w-auto gap-2 rounded-md text-sm lg:mx-0',
```

Comment:
I only now noticed that on Figma it's of font size 12px but here we are using 16px.
The link is using the correct font size but not the text. Please make a change.